blog/_posts/2025-04-24-network-isolated-cluster.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Network Isolated Cluster"
+description: "Use network isolated AKS clusters to restrict outbound traffic at a glance"
+date: 2025-04-24
+author: Charles Wu
+categories: 
+- security
+---
+
+# Network isolated Azure Kubernetes Service (AKS) cluster
+
+By default, standard SKU Azure Kubernetes Service (AKS) clusters have unrestricted outbound internet access. This level of network access allows nodes and services you run to access external resources as needed. For those organizations who have strict security and compliance requirements, to regulate and control egress (outbound) network traffic from a cluster is a must have for the purpose of eliminating data exfiltration.
```

Comment:
May reword "For those organizations who have strict security and compliance requirements, to regulate and control egress (outbound) network traffic from a cluster is a must have for the purpose of eliminating data exfiltration." to something more like "For organizations with  strict security and compliance requirements, regulating and controlling egress (outbound) network traffic from a cluster is a must have for the purpose of eliminating data exfiltration."

Reply:
Reads better, incorporated